lib/matplotlib/axes/_base.py

```diff
@@ -2993,24 +2993,39 @@ def set_xticks(self, ticks, minor=False):
         """
         Set the x ticks with list of *ticks*
 
-        ACCEPTS: sequence of floats
+        Parameters
+        ----------
+        ticks : sequence
+            Sequence of ticks
```

Comment:
I would just say list... and list of floats (I guess?), as it's not a list of Tick objects (unless I got this completely wrong).

Reply:
Could be a list of datetimes if it's a date axis?

Reply:
Possibly, no idea.
But "ticks" without further qualification definitely sounds like "Tick objects" to me.